<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tips</class>
 <widget class="QDialog" name="Tips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>75</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0,0);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>68</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>68</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;
background-color: rgb(160, 160, 160);
border: 6px solid rgb(121, 121, 121);
border-radius:15px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="achievementTitleLabel">
        <property name="minimumSize">
         <size>
          <width>354</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>354</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0;
border-radius:0;
color: rgb(85, 85, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel已达成</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="iconLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0;
color:white;
border-radius:0;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="achievementLabel">
        <property name="minimumSize">
         <size>
          <width>354</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>354</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0;
border-radius:0;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel整装上阵</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
